docs/services/data/configure-data-query.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Configure Data Query"
+linkTitle: "Configure Data Query"
+description: "Configure data query to query synced data directly in the cloud."
+weight: 35
+type: "docs"
+tags: ["data management", "cloud", "query"]
+# SME: Aaron Casas
+---
+
+Configure data query to be able to directly query captured data in the Viam cloud using the [MongoDB Query Language (MQL)](https://www.mongodb.com/docs/manual/tutorial/query-documents/).
+
+## Requirements
+
+Before you can configure [data query](../#data-query), you must [add the data management service](/services/data/configure-data-capture/#add-the-data-management-service) and [configure cloud sync](/services/data/configure-cloud-sync/).
+
+## Configure data query
+
+Once your smart machine has synced captured data to the Viam app, you can configure data query using the Viam CLI:
+
+1. If you haven't already, [install the Viam CLI](/manage/cli/#install) and [authenticate](/manage/cli/#authenticate) to Viam.
+
+1. Find your organization ID by running the following command, or from your organization's **Settings** page in [the Viam App](https://app.viam.com/):
+
+   ```sh {class="line-numbers linkable-line-numbers"}
+   viam organizations list
+   ```
+
+1. Run the following command to determine the hostname for your organization's MongoDB [Atlas Data Federation](https://www.mongodb.com/docs/atlas/data-federation/overview/) instance, which is where your smart machine's synced data is stored.
+   Provide your organizations `org-id` from step 2:
+
+   ```sh {class="line-numbers linkable-line-numbers"}
+   viam data database hostname --org-id=<YOUR-ORGANIZATION-ID>
+   ```
+
+   This command returns the `hostname` of your Viam organization's assigned MongoDB Atlas instance.
+
+1. Then, configure a new database user for that database instance.
+   Provide your organization's `org-id` from step 2, and a desired new password for your database user.
+
+   ```sh {class="line-numbers linkable-line-numbers"}
+   viam data database configure --org-id=<YOUR-ORGANIZATION-ID> --password=<NEW-DBUSER-PASSWORD>
+   ```
+
+   This command creates a new database `user` for your organization's MongoDB Atlas instance.
+
+For more information, see the documentation for the [Viam CLI `database` command](/manage/cli/#data).
+
+## Query data using MQL
```

Comment:
I would probably mention that in this method of connecting directly to MongoDB, you can still use SQL queries by embedding them as $SQL in the Mongo query (example [here](https://www.mongodb.com/docs/atlas/data-federation/query/sql/shell/connect/#aggregation-pipeline-stage-syntax)).

Additionally, would soon (can be a fast-follow once it's in prod and not just in the deployment) add the main flow here that you can just input your SQL/MQL query into the app UI in order to query, and then have this be an alternative method if you want to connect directly to MongoDB.